### DISCUSSION TITLE
Another way to query with parameters

### DISCUSSION THREAD

**Discussion by marcopolooo** [jackc/pgx#1965] [Q&A] [answered]
We know to make query with parameters, we write code like this:

repo.Pgx.QueryRow(ctx, query, ...arg)

So, there is another way to query with named parameter? Like 
```
SELECT * FROM USERS WHERE email = :email and password :password
``` 

Thanks in advance

**Comment by jackc** [comment, marked as answer]
See https://pkg.go.dev/github.com/jackc/pgx/v5#NamedArgs.

**Reply by marcopolooo** [reply]
Thank you very much!
